templates/jdk21/resources/home/liferay/.bashrc

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
@Wykenz, can't we reuse this same file presented in the folder `templates/jdk11/resources`?

Reply:
@natocesarrego actually I don't know if we want to keep the jdk11 folder or not because in the ticket there is this line "Lastly, probably the [code](https://github.com/liferay/liferay-docker/blob/master/templates/jdk11/Dockerfile) should be modified to implement this requested change."  Here "modified" means update the existing jdk11 files to jdk21 or have both in the repository? That is why I made it next to the existing one so if we need only the jdk21 then I can delete the jdk11 folder, but if we need both then I change the jdk21 according to it.

Reply:
Let's keep both for now